apps/dashboard/pkg/migration/conversion/v0.go

```diff
@@ -23,7 +28,19 @@ func Convert_V0_to_V1(in *dashv0.Dashboard, out *dashv1.Dashboard, scope convers
 		},
 	}
 
-	if err := migration.Migrate(out.Spec.Object, schemaversion.LATEST_VERSION); err != nil {
+	// Hack, generate "request" context with namespace info injected,
+	// that migrate.Migrate will use
+	ctx := request.WithNamespace(context.Background(), in.GetNamespace())
+	nsInfo, err := types.ParseNamespace(in.GetNamespace())
+	if err != nil {
+		out.Status.Conversion.Failed = true
+		out.Status.Conversion.Error = ptr.To(err.Error())
+		return nil
+	}
+
+	ctx, _ = identity.WithServiceIdentity(ctx, nsInfo.OrgID)
```

Comment:
```suggestion
	// a background service identity is used here because the user who is reading the specific dashboard
	// may not have access to all the datasources in the dashboard, but the migration still needs to take place
	// in order to be able to convert between k8s versions (so that we have a guaranteed structure to convert between)
	ctx, _ = identity.WithServiceIdentity(ctx, nsInfo.OrgID)
```